pkg/guard-gate/gate.go

```diff
@@ -107,8 +107,9 @@ func (p *plug) ApproveResponse(req *http.Request, resp *http.Response) (*http.Re
 	s.screenResponseBody(resp)
 	s.screenEnvelop()
 	if p.gateState.shouldBlock() && (s.hasAlert() || p.gateState.hasAlert()) {
-		s.cancel()
 		p.gateState.addStat("BlockOnResponse")
+		pi.Log.Debugf("*** Cancel *** during ApproveResponse")
```

Comment:
nit: Shouldn't it be better called "Blocked" instead of "Cancel" as this seems more aligned with the rest of the API (canceling is more the technical way how to achieve blocking)